lib/vlocitycli.js

```diff
@@ -779,3 +785,12 @@ VlocityCLI.prototype.runTestJob = async function(dataPacksJobData) {
     
     return { message: message, status: 'success', action: 'Test Job' };
 }
+
+VlocityCLI.prototype.updateAnalytics = function (eventAction) {
```

Comment:
There still needs to be a way to disable the analytics from the jobInfo and a command line argument.